golang/cosmos/app/upgrade.go

```diff
@@ -33,6 +33,9 @@ func isFirstTimeUpgradeOfThisVersion(app *GaiaApp, ctx sdk.Context) bool {
 // price feed upgrade and associated changes to scaledPriceAuthority and
 // vaultManager.
 func upgradePriceFeedCoreProposalSteps(upgradeName string) ([]vm.CoreProposalStep, error) {
+	isThisUpgrade := func(expectedUpgradeName string) bool {
+		return upgradeName == expectedUpgradeName
```

Comment:
Nice, can we also panic if `expectedUpgradeName` doesn't exist in `upgradeNamesOfThisVersion`, it's so easy to fat finger these names without enum typing